app/__init__.py

```diff
@@ -1,7 +1,27 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy 
+from flask_migrate import Migrate
+
+db = SQLAlchemy()
+migrate = Migrate()
+
+DATABASE_CONNECTION_STRING = 'postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development'
```

Comment:
I suggest that you should use an environment variable for the connection string instead of leaving the string hard-coded here in the `__init__.py` file.